src/QuantumGravityBridge.sol

```diff
@@ -314,24 +310,26 @@ contract QuantumGravityBridge is IDAOracle {
     ///
     /// The data tuple root that is signed over is domain separated as per
     /// `domainSeparateDataRootTupleRoot`.
-    /// @param _nonce The Celestia block height up to which the data root tuple
-    /// root commits to.
+    /// @param _newNonce The new event nonce.
+    /// @param _validatorSetNonce The nonce of the latest update to the
+    /// validator set.
     /// @param _dataRootTupleRoot The Merkle root of data root tuples.
     /// @param _currentValidatorSet The current validator set.
     /// @param _sigs Signatures.
     function submitDataRootTupleRoot(
-        uint256 _nonce,
+        uint256 _newNonce,
+        uint256 _validatorSetNonce,
```

Comment:
Rather than passing in the latest validator set nonce as a parameter, this could be saved in storage. Opinions either way @evan-forbes @SweeXordious?

Reply:
I'm okey either way.

Reply:
same. How much gas would that cost? It would only save a few queries for the relayer, and not that many people will be running relayers I don't think, so probably not worth it add storage me thinks.

Reply:
It would be more expensive in every way to use storage.